### ISSUE TITLE
Reg / Login

### ISSUE THREAD

**Issue by cziem** [phavor/cheetbook#4]
Build out the registration and login forms

**Comment by harshil1712** [comment]
I would like to take this one. Do you mind if I use material ui?

**Comment by cziem** [comment]
Thanks for your interest @harshil1712 

Have you seen the screens? 
![reg](https://github.com/phavor/cheetbook/blob/staging/client/src/images/Reg.png)

You can do that material ui or just css(scss)

But the height of the reg form should be reduced

**Comment by harshil1712** [comment]
@phavor yeah I have seen the screens. I will use material ui and soon send a PR for review

**Comment by cziem** [comment]
No problems @harshil1712

**Comment by cziem** [comment]
No problems @harshil1712
